fury/data/fetcher.py

```diff
@@ -701,9 +721,18 @@ def list_gltf_sample_models():
         Lists the name of glTF sample from
         https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0
     """
-    models = urlopen(GLTF_DATA_URL).read()
-    models = json.loads(models)
-    model_names = [model['name'] for model in models if model['size'] == 0]
+    DATA_DIR = pjoin(dirname(__file__), 'files')
+    with open(pjoin(DATA_DIR, 'KhronosGltfSamples.json'), 'r') as f:
+        models = json.loads(f.read())
+    models = models.keys()
+    model_modes = [model.split('/')[0] for model in models]
```

Comment:
same comment as above:

I am surprised that this line works on `Windows`.

@ganimtron-10 and @m-agour, can you show me the result (return value) of `fetch_gltf(...)`?